lib/internal/worker.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
 
 /* global SharedArrayBuffer */
+/* global clearInterval */
+/* global setInterval */
```

Comment:
We should probably get these from `require('timers')`, like we also enforce for `Buffer`?